spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/DefaultSubscriberFactory.java

```diff
@@ -156,12 +159,22 @@ public void setApiClock(ApiClock apiClock) {
 	/**
 	 * Set the retry settings for the generated subscriber stubs.
 	 * @param subscriberStubRetrySettings parameters for retrying pull requests when they fail,
-	 * including jitter logic, timeout, and exponential backoff
+	 *                                    including jitter logic, timeout, and exponential backoff
 	 */
 	public void setSubscriberStubRetrySettings(RetrySettings subscriberStubRetrySettings) {
 		this.subscriberStubRetrySettings = subscriberStubRetrySettings;
 	}
 
+	/**
+	 * Set the (n)acknowledgement of messages to be asynchronous
+	 * @param asyncAck
+	 *
```

Comment:
I think there should be some option in the IDE to disable JavaDocs auto-formating and, therefore, there won't be this extra blank line if you don't add it explicitly.

Thanks for understanding.

Reply:
Fixed.